tests/yoti_common/anchor.processor.spec.js

```diff
@@ -119,6 +118,33 @@ describe('anchorProcessor', () => {
         expect(expectedCertificate).to.equal(certificates[0].signatureOid);
       });
     });
+
+    context('when processing multiple anchors', () => {
+      const dlSourceAnchor = fs.readFileSync('./tests/sample-data/yoti-common/dl-source-anchor.txt', 'utf8');
+      const verifierAnchor = fs.readFileSync('./tests/sample-data/yoti-common/verifier-anchor.txt', 'utf8');
+      const unknownAnchor = fs.readFileSync('./tests/sample-data/yoti-common/unknown-anchor.txt', 'utf8');
+
+      const anchors = AnchorProcessor.process([
+        parseAnchorData(dlSourceAnchor),
+        parseAnchorData(verifierAnchor),
+        parseAnchorData(unknownAnchor),
+      ]);
+
+      it('should return 1 source anchor', () => {
+        expect(anchors.sources).to.be.length(1);
+        expect(anchors.sources[0].getType()).to.equal('SOURCE');
+      });
+
+      it('should return 1 verifier anchor', () => {
+        expect(anchors.verifiers).to.be.length(1);
+        expect(anchors.verifiers[0].getType()).to.equal('VERIFIER');
+      });
+
+      it('should return 1 unknown anchor', () => {
+        expect(anchors.unknown).to.be.length(1);
+        expect(anchors.unknown[0].getType()).to.equal('UNKNOWN');
+      });
```

Comment:
No harm in checking that there are 3 in total here as well is there?

Reply:
I've added a check in the attribute test in 5d285b0, as this test returns a key-value object, rather than a flat array of anchors.